src/locales/locale-en.json

```diff
@@ -6,7 +6,8 @@
     "WARNING": "Warning",
     "BETA_WARNING": "This website is under active development and everything is subject to change. We can't guarantee for the safety and privacy of the data you enter (but we try our best).",
     "CONNECTION_INTERRUPTED": "Error: connection interrupted!",
-    "SERVER_ERROR": "Server error"
+    "SERVER_ERROR": "Server error",
+    "GLOBAL.SEARCH_NOT_FOUND": "Not Found"
```

Comment:
`GLOBAL` is duplicate here, since it's already its namespace. You should just name it  `SEARCH_NOT_FOUND`, then use it in the code with `GLOBAL.SEARCH_NOT_FOUND`.

Maybe `NOT_FOUND` instead of `SEARCH_NOT_FOUND` is also better, since it's a pretty generic expression.